doc/nrf/device_guides/nrf70/nrf7002eb_54h20dk_gs.rst

```diff
@@ -0,0 +1,81 @@
+:orphan:
+
+.. _ug_nrf7002eb_nrf54h20dk_gs:
+
+Getting started with nRF7002 EB for nRF54H20 DK
+###############################################
+
+.. contents::
+   :local:
+   :depth: 4
+
+This section provides information about how to use the nRF7002 :term:`Expansion Board (EB)` (PCA63561) to provide Wi-Fi® connectivity to the :ref:`zephyr:nrf54h20dk_nrf54h20`.
+The changes mentioned are specific to the nRF54H20 DK.
+See the :ref:`nRF7002 EB <ug_nrf7002eb_gs>` documentation for more details on building and programming with the nRF7002 EB.
+
+Overview
+********
+
+The nRF7002 EB is a shield that can be used with the nRF54H20 DK to add support for Wi-Fi.
+
+Limitations
+===========
+
+The Wi-Fi support is experimental and has the following limitations:
+
+* It only supports STA mode.
+* It only allows Wi-Fi open security.
+* It is only suitable for low-throughput applications.
+
+Pin mapping for nRF54H20 DK
+===========================
+
+The pin mapping for the nRF54H20 DK is as follows:
+
++-----------------------+-----------------------+-----------------------------------------------+
+| nRF700x pin name (EB) | nRF54H20 DK pins      | Function                                      |
```

Comment:
```suggestion
| nRF70 Series pin name (EB) | nRF54H20 DK pins      | Function                                      |
```

Reply:
@bama-nordic and @busineni has specifically suggested this change, so, will leave it to them to take a final call.